### PR TITLE
Issue 325: [website] how to manage documentation between releases

### PR DESCRIPTION
**Pull request by sijie**
Descriptions of the changes in this PR:

- move `docs` to `docs/latest`
- add a script to proceed release steps
   - copy `docs/latest` to `docs/{version}`
   - in docs/{version}, replace versions
   - bump versions in `_config.yml`
   - generate placeholders for releaseNotes

### CODE REVIEW

**Comment by sijie** [comment]
https://sijie.github.io/bookkeeper-staging-site/ is how it looks like after this change.

File: site/_config.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 title: Apache BookKeeper
-subtitle: A replicated log service
+subtitle: A scalable, fault tolerant and low latency storage serivce optimized for realtime workloads
```
Comment (lucperkins) [inline site/_config.yml]: A scalable, fault-tolerant, and low-latency storage service optimized for real-time workloads

Comment (sijie) [inline site/_config.yml]: addressed the comments

**Pull request closed without merging**